handlers/facebook/facebook.go

```diff
@@ -182,7 +182,12 @@ type moPayload struct {
 				Attachments []struct {
 					Type    string `json:"type"`
 					Payload *struct {
-						URL string `json:"url"`
+						URL         string `json:"url"`
+						StickerID   int64  `json:"sticker_id"`
```

Comment:
newer API has sticker ID in the attachment so I added that here well and we'll support both cases, and the older taking over if present, so we loop on attachment before checking the stickerID outside template and we'll set the later as text in case it is present